crates/rust-analyzer/src/main_loop.rs

```diff
@@ -624,7 +661,17 @@ fn on_notification(
         Err(not) => not,
     };
     let not = match notification_cast::<req::DidChangeConfiguration>(not) {
-        Ok(_params) => {
```

Comment:
Hm, yeah, given that we already have no-op handler to avoid errors, I think clients *do* send this notification already?

Reply:
It did not trigger for me when I tried to change the settings on `master`, so it looks like they _don't_.

But I find this surprising also, I would have expected it to be working by default.